src/Timeline.js

```diff
@@ -179,14 +175,44 @@ export default class Timeline extends Axis {
     this._brushGroup.selectAll(".overlay")
       .attr("cursor", this._brushing ? "crosshair" : "pointer");
 
-    this._brushGroup.selectAll(".selection")
+    const yTransform = this._height / 2 - this._buttonHeight / 2;
```

Comment:
the variables `yTransform` should only be calculated inside the `if` statement where it is used